core/dbt/deprecations.py

```diff
@@ -87,6 +87,16 @@ class AdapterDeprecationWarning(DBTDeprecation):
     deprecations[dep.name] = dep
 
 
+class MetricAttributesRenamed(DBTDeprecation):
+    _name = "metric-attr-renamed"
+    _description = """\
+dbt-core v1.3 renamed attributes for metrics:
+\n  'sql'              -> 'expression'
+\n  'type'             -> 'calculation_method'
+\n  'type: expression' -> 'calculation_method: derived'
+    """
```

Comment:
...and the deprecation message here

Reply:
Is it possible to include the specific metric name in the deprecation message? Something like:

    _description = """\
dbt-core v1.3 renamed attributes for metrics:
\n  'sql'              -> 'expression'
\n  'type'             -> 'calculation_method'
\n  'type: expression' -> 'calculation_method: derived'
\n 
\n. The now old metric parameters will be fully deprecated in version 1.4.0. Please remove them from the metric definition of metric {{metric_name}} 
"""

Reply:
yes for sure!